CareKit/CareKit/iOS/Task/SynchronizedSliderTaskView.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  OCKSliderButton.swift
+//
+//
+//  Created by Dylan Li on 7/21/20.
+//  Copyright © 2020 NetReconLab. All rights reserved.
+//
+
+#if !os(watchOS)
+
+import CareKitStore
+import CareKitUI
+import Foundation
+import SwiftUI
+
+@available(iOS 14.0, *)
+public struct SynchronizedSliderTaskView<Controller: OCKSliderTaskController, SliderTaskView: View>: View {
```

Comment:
You can use the `SynchronizedTaskView ` instead of creating a new struct specifically for the slider task view.